src/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { normalizePath } from "obsidian";
+
+interface ParsedPath {
+  /** The full directory path such as '/home/user/dir' or 'folder/sub' */
+  dir: string;
+  /** The file name without extension */
+  name: string;
+}
+
+export const path = {
+  /**
+   * Parses a path string into directory and file name
+   * Supports '/foo/bar/file' or '\foo\bar\file'
+   */
+  parse(path: string): ParsedPath {
```

Comment:
May be we can use builtin node.js functions?
https://nodejs.org/api/path.html#pathdirnamepath

Reply:
I tried that initially but it only worked on desktop.

Reply:
I was getting an error in the IOS app when I tried to use built in node packages.

Reply:
```js 
import  * as path from 'path'

...

path.join("strings"
```

Results in an error on the IOS app.  Not sure why.

Reply:
Okay :(